### PR TITLE
Adding macro project-bundle-migration

### PR DESCRIPTION
**Pull request by pmasiphelps**
Moving the Project Bundle Migration macro from dku-contrib-private to the public repo.

### CODE REVIEW

**Comment by ThomasCabrol** [comment]
As a general comment - it may be great to rename this plugin to something like "dataiku-project-bundle-migration" to make it more specific to Dataiku

File: project-bundle-migration/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+To do this, click on the "Bundles" tab of the project, then "More", then "Configure Bundles Content".
+
+![Configure_bundles_content](doc/Configure_bundles_content.png)
+
+Add all uploaded files, server_filesystem files, and deployed models using the +Add button.
+
+![Add_objects_to_bundle](doc/Add_objects_to_bundle.png)
+
+3. Create a Global API key on the target automation instance, which should be used as an input for this macro.
```
Comment (ThomasCabrol) [inline project-bundle-migration/README.md]: The README could be more complete and include an "how to use" section explaining more in details what it does, how to use it, and a few screenshot. Maybe also add a comment on the fact that it requires to have an Automation instance :)

File: project-bundle-migration/plugin.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "id" : "Project_Bundle_Migration",
```
Comment (ThomasCabrol) [inline project-bundle-migration/plugin.json]: I would use a more "standard" ID sticking with the plugin name, so "dataiku-project-bundle-migration" for instance

File: project-bundle-migration/README.md (hunk 1 above)
Comment (pmasiphelps) [reply project-bundle-migration/README.md]: Yes, I added a few additional lines to Overview section of the README

**Pull request merged**